tests/regression/test_regression.py

```diff
@@ -132,3 +132,51 @@ def test_regression(
             fxt_num_repeat=fxt_num_repeat,
             tmpdir=tmpdir,
         )
+
+class TestMultilabelCls(BaseTest):
+    # Test case parametrization for model
+    MODEL_TEST_CASES = [  # noqa: RUF012
+        ModelTestCase(task="multi_label_classification", name="efficientnet_b0_light"),
```

Comment:
It seems not aligned with the real path `recipe/multilabel_classification`.

Reply:
Done, thanks. https://github.com/openvinotoolkit/training_extensions/pull/2739/commits/b6beac6aeb95277e4a8a95eb2026ea78f43f8687